lib/main-menu.js

```diff
@@ -258,7 +258,7 @@ const view = {
   submenu: [
     {
       label: 'Reload',
-      accelerator: 'CommandOrControl+R',
+      accelerator: 'CommandOrControl+Shift+R',
```

Comment:
There is no reason to change the shortcut for all users. This change should only be made for users of vim keybinds.

Reply:
For me it seemed like a development feature. I don't see a situation when somebody would want to reload the app.

Reply:
Reloading the app is used similarly as the refresh button in your browser. It updates any files that might have changed due to external (cloud sync) changes. Any changes to configs that might have failed to propagate (or were synced) will also only be applied after a reload of the app.

Since the conflict only exists for users of vim keybinds, the issue only needs to solved there.